MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,1187 @@
+                details.Add(new EmployeeInfo() { ID = 9, Name = "Aaron"  });
+
+                return details;
+            }
+        return new List<string>();
+        }
+    }
+
+    public class ContactInfo
+    {
+        [Display(Name ="First Name")]
+        public String FirstName { get; set; } 
+        public string EmployeeDetails { get; set; }
+    }
+
+    public class EmployeeInfo
+    {
+        public int ID { get; set; }
+        public string Name { get; set; }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+`Image`
+
+## RadioGroup editor
+
+In the [RadioGroup](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/radiobutton?view=net-maui-7.0) editor, the [RadioButton](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/radiobutton?view=net-maui-7.0) control is loaded.
+
+The `items` for [RadioButton](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/radiobutton?view=net-maui-7.0) is generated for `enum` and `List` data type properties. In order to add [RadioButton](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/radiobutton?view=net-maui-7.0) editor in the DataForm, you need to register editor as DataFormEditorType.RadioGroup for the required property by using the `RegisterEditor` method.
+
+**Support for enum data type**
+
+For `enum` data type property, [RadioButon](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/radiobutton?view=net-maui-7.0) `items` will be added based on specified property enum values.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.RegisterEditor("Phone", DataFormEditorType.RadioGroup);
+
+    public Numbers Phone  { get; set; }
+
+    public enum Numbers
+    {
+	    Home,
+	    Work,
+	    Other
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+**Support for List data type**
+
+For `List` data type property, you have to set the `ItemsSource` by using the `IDataFormSourceProvider`, based on that [RadioButton](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/radiobutton?view=net-maui-7.0) `items` will be added.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.RegisterEditor("Phone", DataFormEditorType.RadioGroup);
+    dataForm.ItemsSourceProvider = new DataFormItemsSourceProvider();
+
+    public string Phone  { get; set; }
+
+    public class DataFormItemsSourceProvider : IDataFormSourceProvider
+    {
+         public object GetSource(string sourceName)
+        {
+		    if(sourceName == "Phone")
+		    {
+                List<string> list = new List<string>() {"Home", "Work", "Other"};
+                return list;
+		    }
+		    return new List<string>();
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+`Image`
+	
+## Custom editor
+
+The custom editor can be added to DataForm by inheriting the `IDataFormEditor` class for business models. You can create custom editor using implement all method in IDataFormEditor. 
```

Comment:
- The custom editor can be added to DataForm by inheriting the `IDataFormEditor` class for business models. You can create a custom editor by implementing all methods in `IDataFormEditor`.

Reply:
updated